meson.build

```diff
@@ -0,0 +1,150 @@
+project('GMime',
+	['c'],
+	license: 'GPLv2',
+    version: '3.2.13',
+    meson_version: '>=1.0.0',
+    default_options : [
+    ])
+
+gmime_version = meson.project_version()
+ver_arr = gmime_version.split('.')
+
+GMIME_MAJOR_VERSION=ver_arr[0].to_int()
+GMIME_MINOR_VERSION=ver_arr[1].to_int()
+GMIME_MICRO_VERSION=ver_arr[2].to_int()
+GMIME_BINARY_AGE= 100 * GMIME_MINOR_VERSION + GMIME_MICRO_VERSION
+GMIME_INTERFACE_AGE=0
+
+gmime_package = 'gmime-@0@.0'.format(GMIME_MAJOR_VERSION)
+
+add_global_arguments('-DHAVE_CONFIG_H=1', language : 'c')
+
+iconv = dependency('iconv', required : false, method: get_option('iconv'))
+libidn2 = dependency('libidn2', version: '>=2.0.0', required : false)
+gpgme = dependency('gpgme', version: '>=1.6.0', required: get_option('crypto'))
+glib =  dependency('glib-2.0', version: '>=2.68')
+gobject = dependency('gobject-2.0')
+deps = [
+  dependency('zlib', version: '>=1.2.5.2'),
+  glib,
+  gpgme,
+  gobject,
+  dependency('gio-2.0'),
+  libidn2,
+  iconv,
+]
+
+gmime_prefix = get_option('prefix')
+gmime_datadir = join_paths(gmime_prefix, get_option('datadir'))
```

Comment:
This doesn't appear to ever be used, nor can I find anything in the source code or autotools that it would correspond to.

Reply:
Ah. Those are used for the docs, which are not included in this PR. Removing.